fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -612,7 +612,12 @@ private void addLinkAndOptionsHttpHeaders() {
             servletResponse.addHeader("Accept-Patch", contentTypeSPARQLUpdate);
 
         } else if (resource() instanceof Container) {
-            options = "MOVE,COPY,DELETE,POST,HEAD,GET,PUT,PATCH,OPTIONS";
+            // MOVE/COPY/DELETE operations are not available on a RepositoryRoot resource
+            if (resource().getContainer() == null) {
+                options = "POST,HEAD,GET,PUT,PATCH,OPTIONS";
```

Comment:
It's certainly not _forbidden_, but it's not supported, so it seems wrong to advertise that it is supported.